### ISSUE TITLE
Don't assign incremental IDs to scan results

### ISSUE THREAD

**Issue by JonasDralle** [PrivacyScore/PrivacyScore#43]
https://privacyscore.org/site/ *
	
By returning an incremental scan result ID you expose various information about your users.

If someone submits multiple pages to scan they get adjacent IDs => identifiable  
The order of the scan requests also expose spread of your service.

For example the first scan result is of uni hamburg (www.uni-hamburg.de/)  
https://privacyscore.org/site/1/

followed by various other pages (including subdomains and printing services of uni hamburg, apparently you have studied CS there).

Some companies (i. e. Facebook) have elaborate social graphs that could reconstruct the exact flow of webservice recommendations because of the icnremental ID.

**Comment by JonasDralle** [comment]
thinking of which it might make more sense to have

    https://privacyscore.org/site/github.com/

as URL instead of

    https://privacyscore.org/site/18489/

**Comment by muelli** [comment]
yeah, I'm in favour of that approach.  We don't mind the exposure part though.
Anyway, sounds like https://github.com/PrivacyScore/PrivacyScore/issues/18, in parts at least.

**Comment by JonasDralle** [comment]
PS implements an open data policy which is great so exposure of all datasets isn't so bad.  
Yet it doesn't favour the privacy of the users if you can see (some domain only known to a few) ; pornhub ; (some domain only known to a few)

Many selfhosters have their own domain which means  
`knowledge of domain`=>`has connection to the hoster`
